src/components/ContainerHeader.vue

```diff
@@ -278,6 +271,13 @@ export default {
                 network.ircClient.join(channelName);
             }
         },
+        nicklistToggle: function nicklistToggle() {
```

Comment:
This should use the function shorthand, `nicklistToggle() {`. Also lets rename it to `toggleNicklist()` so it's consistent with the function name a few lines below